src/OpenLoco/src/Windows/Construction/Construction.h

```diff
@@ -1,5 +1,6 @@
 #pragma once
 
+#include "GameState.h"
```

Comment:
Please move the include to `ConstructionTab.cpp`, so we keep this header relatively light.